lib/android/build.gradle

```diff
@@ -38,7 +38,5 @@ dependencies {
   compile 'com.android.support:support-annotations:25.2.0'
   compile 'com.android.support:design:25.2.0'
   compile 'com.fasterxml.jackson.core:jackson-databind:2.8.3'
-  compile 'com.facebook.fresco:fresco:0.11.0'
```

Comment:
Can you put this in a separate PR?
It'll also conflict with this: https://github.com/airbnb/native-navigation/pull/107